Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.yml

```diff
@@ -157,7 +157,7 @@ script:
       description: Indicator ID.
       type: number
     description: Lookup reputation in the indicators.
-  dockerimage: demisto/taxii2:1.0.0.18446
+  dockerimage: demisto/taxii2:1.0.0.23423
```

Comment:
Please don't change this image here, there is a bug in the new taxii2.

Reply:
This one - https://github.com/demisto/etc/issues/40563

Reply:
I'll add the field to YML

Reply:
this is insufficient 
please open an dev-task to track the taxii2 version and bump the docker image once possible

Reply:
I'll make sure we are handling this correctly. 
Up until now it was just for base images.